Files/ViewModels/ItemViewModel.cs

```diff
@@ -399,19 +398,17 @@ private async void Connection_RequestReceived(object sender, Dictionary<string,
             }
         }
 
-        public async Task CancelLoadAndClearFiles()
+        public void CancelLoadAndClearFiles()
         {
-            await Task.Run(() =>
+            Debug.WriteLine("CancelLoadAndClearFiles");
+            CloseWatcher();
+            if (IsLoadingItems)
             {
-                Debug.WriteLine("CancelLoadAndClearFiles");
-                CloseWatcher();
-                if (IsLoadingItems)
-                {
-                    addFilesCTS.Cancel();
-                }
-                CancelExtendedPropertiesLoading();
-                filesAndFolders.Clear();
-            });
+                addFilesCTS.Cancel();
+            }
+            CancelExtendedPropertiesLoading();
+            filesAndFolders.Clear();
+            FilesAndFolders.Clear();
```

Comment:
Added `FilesAndFolders.Clear()` here so the UI list is cleared when a tab is closed/when the Dispose method is called.

Reply:
Good catch! This one is quite impactful🎉🎉

Reply:
If `filesAndFolders` is a part of `FilesAndFolders` then we could remove that redundant call to former one.